plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/VifDriverBase.java

```diff
@@ -41,6 +43,11 @@ public void configure(Map<String, Object> params) throws ConfigurationException
         _pifs = (Map<String, String>)params.get("libvirt.host.pifs");
     }
 
+    protected String getControlCidr(String defaultValue, Map<String, Object> params) {
```

Comment:
Almost looks good. You aren't using the params variable anywhere, right?

Reply:
sorry, I do not give this pr proper tlc (bit detached fromit)